fk/lisjp/src/main/java/se/inera/intyg/common/lisjp/v1/model/converter/InternalToCertificate.java

```diff
@@ -1097,6 +1102,7 @@ private static boolean hasRenewalRelation(Relation relation) {
             return Collections.emptyList();
         }
         return sickLeaves.stream()
+            .filter(item -> item.getPeriod().isValid())
```

Comment:
Could item.getPeriod() be null?

Reply:
Good question! I will look into it before merging!

Reply:
It was easy to do with a unit test so it could probably happen.

Reply:
Fixed